android/app/src/main/java/app/organicmaps/widget/placepage/PlacePageController.java

```diff
@@ -283,6 +283,9 @@ private void setPeekHeight()
    */
   private void animatePeekHeight(int peekHeight)
   {
+    if (mCurrentWindowInsets == null) {
```

Comment:
Thanks! 

What is the case/state when it is null? Is it a valid state? How does the app get into this state?

Reply:
I don't really know to be honest. I made a video. Basically having a location open and clicking somewhere close of that point _sometimes_ causes it to crash, I suspect that it crashes right after the first start of the app, but if you clicked e.g. further away from the POI and then try to reproduce it, it doesn't happen anymore.
Video.
[Screen_recording_20241206_221639.webm](https://github.com/user-attachments/assets/63f66d36-a52c-415a-a004-a3cfcba797ee)

All I did was select a POI and click on the map again close to it (so that it would select it again) right after starting the app

Reply:
Thanks for the video! @strekha maybe you have some ideas why `mCurrentWindowInsets` is nill here? Is a more correct fix possible?

Reply:
@biodranik there are a lot of such checks in this file, so adding one here looks right

Reply:
Am I correctly hearing that it is about repeating a "safe null check" pattern without clearly understanding _why_ it is null?

Reply:
@biodranik, I can't comment on this file, because the work to support window insets was done by someone else, not me. I don't understand all the usecases and lifecycle of this screen